Error when building projects/example

```c++
Scanning dependencies of target example
[ 96%] Building CXX object projects/example/CMakeFiles/example.dir/main.cpp.o
[ 96%] Built target example_utility
/home/developer/local-pathfinding/projects/example/main.cpp:9:11: error: no matching constructor for initialization of 'Planner'
  Planner p(0, std::vector<Obstacle>(), 0, 0, nullptr, nullptr);
          ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/developer/local-pathfinding/src/planning/Planner.h:22:3: note: candidate constructor not viable: no known conversion from 'nullptr_t' to 'const ompl::base::ScopedState<ompl::base::SE2StateSpace>' for 5th argument
  Planner(double windDirection,
  ^
/home/developer/local-pathfinding/src/planning/Planner.h:20:7: note: candidate constructor (the implicit copy constructor) not viable: requires 1 argument, but 6 were provided
class Planner {
      ^
/home/developer/local-pathfinding/src/planning/Planner.h:20:7: note: candidate constructor (the implicit move constructor) not viable: requires 1 argument, but 6 were provided
1 error generated.
projects/example/CMakeFiles/example.dir/build.make:62: recipe for target 'projects/example/CMakeFiles/example.dir/main.cpp.o' failed
```

Build succeeds after commenting out the following line:
https://github.com/UBCSailbot/local-pathfinding/blob/036f49be2efa051329083ef3b9d4156b25eb5802/projects/CMakeLists.txt#L1

--- a/projects/example/main.cpp
+++ b/projects/example/main.cpp
@@ -6,7 +6,16 @@
 
 int main(int, char *[]) {
   std::cout << "OMPL version: " << OMPL_VERSION << std::endl;
-  Planner p(0, std::vector<Obstacle>(), 0, 0, nullptr, nullptr);
+
+  ompl::base::StateSpacePtr space(Planner::getStateSpace(0, 5));
+  ompl::base::ScopedState<ompl::base::CompoundStateSpace> start(space);
+  start->as<ompl::base::SE2StateSpace::StateType>(0)->setXY(0, 1);
+  start->as<ompl::base::SE2StateSpace::StateType>(0)->setYaw(M_PI / 4);
+  start->as<ompl::base::RealVectorStateSpace::StateType>(1)->values[0] = 0;
+
+  BoatPosition goal(Coordinate(4, 4), M_PI / 4);
+
+  Planner p(0, std::vector<Obstacle>(), -2, 7, start, goal);
 
   p.printSetup();
   auto solved = p.Solve(1.0);

